components/builder-api-client/src/builder.rs

```diff
@@ -399,28 +409,27 @@ impl BuilderAPIProvider for BuilderAPIClient {
                                    token: &str,
                                    promote: bool)
                                    -> Result<()> {
+        debug!("Promote/demote for group: {}, channel: {}",
+               group_id, channel);
```

Comment:
It would be useful to include the concrete operation being performed here (either promote or demote), rather than being ambiguous.

Reply:
Ha, yes...